app/models/veteran_onboarding.rb

```diff
@@ -17,16 +17,42 @@ class VeteranOnboarding < ApplicationRecord
   belongs_to :user_account, primary_key: :id, foreign_key: :user_account_uuid, inverse_of: :veteran_onboarding
   validates :user_account, uniqueness: true
 
+  attr_accessor :user
+
   # Determines whether the onboarding flow should be displayed for a veteran.
-  # Currently, this is based solely on the value of the `display_onboarding_flow` attribute.
-  # In the future, additional information (like MPI info) might be taken into account.
+  # Currently, we have two feature toggle checks:
+  # - veteran_onboarding_show_to_newly_onboarded examines settings for veteran_onboarding.onboarding_threshold_days,
+  #   (default 180 days) and examines the user_verification.verified_at attribute
+  # - veteran_onboarding_beta_flow just checks if the feature toggle is enabled for the user
   def show_onboarding_flow_on_login
-    display_onboarding_flow
+    if @user.user_account&.verified?
+      if Flipper.enabled?(:veteran_onboarding_show_to_newly_onboarded, @user)
+        days_since_verification = UserVerification.where(user_account_id: @user.user_account_uuid).map do |uv|
+          (Time.zone.today - uv.verified_at.to_date).to_i
+        end.max
+        threshold_days = Settings.veteran_onboarding&.onboarding_threshold_days || 180
```

Comment:
I don't think it's required to add `veteran_onboarding` to [config/settings.yml](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml) because you have the default set here as `180`. And I saw you mentioned you will be adding the values to the manifest repo, so that will cover it (and the values set there overwrite anything in settings.yml), but I usually see people add their config to Settings, so I'm wondering why you decided not to.

Reply:
I did not add values to Settings because as you said, they will be overwritten by the manifest repo.  To me, as a newbie coming on to VA.gov, I would be confused if the value in `Settings` was for example 90 days, but veterans were seeing the onboarding message at 180 days.  This is just me bringing the baggage of my own experience (Why in the world is the application not respecting the value in `Settings`?  I can see it right there).  

I am happy to add an entry in `Settings` along with a comment that the value could be overwritten by the manifest repo, if you would prefer.

Reply:
That's valid. I'm used to checking config/settings.yml whenever I see a `Settings.`, but I agree that file can be misleading.